ISO-process/ISO-PAS-Explanatory-Report-For-OGF-DFDL.md

```diff
@@ -0,0 +1,353 @@
+
+Not applicable.
+
+#### 7.4.3 Alignment
+_The specification should be aligned with existing JTC 1 standards or ongoing work and thus complement
+existing standards, architectures and style guides. 
+Any conflicts with existing standards, architectures
+and style guides should be made clear and justified._
+
+##### 7.4.3.1 Relationship to Existing Standards
+_a) What International Standards are closely related to the specification and how?_
+
+The DFDL Specification is layered on top of W3C [XML Schema](https://www.w3.org/TR/xmlschema-0/), and 
+the DFDL expression language is based on [W3C XPath 2.0](https://www.w3.org/TR/xpath20/). 
+
+A related standard is ASN.1 ECN, i.e.,  ITU-T Recommendation X.692 | ISO/IEC 8825-3, Information technology – ASN.1 
+encoding rules: Specification of Encoding Control Notation (ECN). 
+ASN1 ECN is an earlier attempt to create a declarative format description system. 
+
+_b) To what International Standards is the proposed specification a natural extension?_
+
+W3C XML Schema and W3C XPath - see above. 
+
+_c) How the specification is related to emerging and ongoing JTC 1 projects?_
+
+JTC 1 SC41 (Internet of Things, Digital Twin) has a clear relationship to cloud/grid computing technology which has
+clear relationship to the mission of the OGF generally and DFDL specifically. 
+
+
+
+##### 7.4.3.2 Adaptability and Migration
+_a) What adaptations (migrations) of either the specification or International Standards would improve
+the relationship between the specification and International Standards?_
+
+Three ISO data format standards, EDIFACT ISO 9735, ISO 8583, and Swift ISO 15022 subset were integral to
+creation and acceptance of the DFDL standard at the OGF level.
+Going forward additional ISO data standards can be described using DFDL schemas.
+
+_b) How much flexibility does the PAS Submitter have?_
+
+The submission is made by the OGF DFDL Working Group.
+
+_c) What are the longer-range plans for new/evolving specifications?_
+
+The DFDL workgroup strives for stability in the specification, and robust compatibility between revisions. 
+
+A standard mechanism is in place for proposing extensions to DFDL. 
+This requires development of a prototype implementation that adheres to the guidelines ensuring 
+that the proposed extension features are clearly identifiable in the text so users do not become 
+dependent on these features accidentally. 
+Experience reports about real use cases and the experience with the feature are required 
+before consideration of incorporation of an extension into a future version of the DFDL standard.   
+
+##### 7.4.3.3 Substitution and Replacement
+_a) What needs exist, if any, to replace an existing International Standard? Rationale?_
+
+Not applicable.
+
+_b) What is the need and feasibility of using only a portion of the specification as an International
+Standard?_
+
+DFDL is a large standard; hence, it explicitly allows _conforming_ subsets, and identifies 
```

Comment:
I'm not sure that's the right answer. The question is asking whether ISO could adopt just part of the DFDL spec. We don't want that. So while you can point out that DFDL allows conforming subsets, we want to recommend that the whole spec is adopted, and is not structured in a way that makes it amenable to adopting only a portion.